src/UmaCtfAdapter.sol

```diff
@@ -203,7 +201,7 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
         if (payouts.length != 2) revert InvalidPayouts();
         if (!_isInitialized(questionData)) revert NotInitialized();
         if (!_isFlagged(questionData)) revert NotFlagged();
-        if (block.timestamp <= questionData.adminResolutionTimestamp) revert SafetyPeriodNotPassed();
+        if (block.timestamp <= questionData.emergencyResolutionTimestamp) revert SafetyPeriodNotPassed();
```

Comment:
Can we make this `<` so that the ER timestamp is the first time that it's allowed to emergencyResolve?